lib/content-services/src/lib/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -33,6 +33,9 @@ describe('Breadcrumb', () => {
     });
     let documentListComponent: DocumentListComponent;
 
+    const getBreadcrumbActionText = (fx: ComponentFixture<BreadcrumbComponent>): string =>
+        fx.debugElement.nativeElement.querySelector('.adf-breadcrumb-item-current').textContent.trim();
```

Comment:
This can be simplified even more. You don't need to pass fx (fixture) as parameter because it is defined in whole describe so you have direct access to it. We often use that directly instead of passing by parameter because these tests are for breadcrumb so you won't have any other fixture so no need for parameter.